qds_sdk/commands.py

```diff
@@ -1347,6 +1347,64 @@ def parse(cls, args):
         v["command_type"] = "DbTapQueryCommand"
         return v
 
+class JupyterNotebookCommand(Command):
+    usage = "jupyternotebookcmd <submit|run> [options]"
+
+    optparser = GentleOptionParser(usage=usage)
+    optparser.add_option("--path", dest="path", help="Path including name of the Jupyter notebook to be run with extension.")
+    optparser.add_option("--cluster-label", dest="label", help="Label of the cluster on which the this command should be run. If this parameter is not specified then label = 'default' is used.")
+    optparser.add_option("--arguments", dest="arguments", help="Valid JSON to be sent to the notebook. Specify the parameters in notebooks and pass the parameter value using the JSON format. key is the parameter's name and value is the parameter's value. Supported types in parameters are string, integer, float and boolean.")
+    optparser.add_option("--macros", dest="macros", help="expressions to expand macros used in query")
+    optparser.add_option("--name", dest="name", help="Assign a name to this query")
+    optparser.add_option("--tags", dest="tags", help="comma-separated list of tags to be associated with the query ( e.g. tag1 tag1,tag2 )")
+    optparser.add_option("--notify", action="store_true", dest="can_notify", default=False, help="sends an email on command completion")
+    optparser.add_option("--timeout", dest="timeout", type="int", help="Timeout for command execution in seconds")
+    optparser.add_option("--retry", dest="retry", choices=['1','2','3'], help="Number of retries for a job")
+    optparser.add_option("--retry-delay", dest="retry_delay", type="int", help="Time interval between the retries when a job fails.")
+    optparser.add_option("--pool", dest="pool", help="Specify the Fairscheduler pool name for the command to use")
```

Comment:
is this supported for Jupyter Notebook commands?

Reply:
I have not explicitly tried this out, but this option is available in Scheduler page for Jupyter Notebook Command so I have this here.